packages/dm-core/src/types.ts

```diff
@@ -132,6 +132,16 @@ export type TInlineRecipeViewConfig = TViewConfig & {
   recipe: TUiRecipe
 }
 
+export type TFileEntity = {
+  type: string
+  name: string
+  author: string
+  date: string
+  size: number
+  filetype: string
+  content?: TLinkReference
```

Comment:
According to SIMOS/File.json, content has type "binary". Is that wrong?

Reply:
I was a bit confused by this. Since I don't know how the binary type is supposed to work (and I could not find any documentation about it), 

from some parts of dmss it looks like if type is "binary", it is interpreted as a storage reference.
![image](https://github.com/equinor/dm-core-packages/assets/69512295/f5629b44-32a2-476e-a752-208f012bd640)

However, from the implementation of the File plugin in dm-core-packages it looks like a link reference is required.
So I dont know if we should use link or storage reference for binary type.

we need to take another look at this once @eoaksnes  is back I think [(will add issue for this)](https://github.com/equinor/dm-core-packages/issues/403)

Reply:
I did try to use a storage reference instead of a link reference in the File plugin, but could not make it work

Reply:
I don't know either. But I don't think it blocks this PR in any way, though. We can ask Eirik Ola next week